lib/ecto_enum.ex

```diff
@@ -79,6 +102,10 @@ defmodule EctoEnum do
           {:ok, @enum_map[int]}
         end
 
+        def load(string) when is_binary(string) do
+          {:ok, String.to_atom(string)}
```

Comment:
Thought the diff would be higher. Anyway, we still should avoid using `String.to_atom/1`, because we would convert any string from the DB to an atom. The only possible values should be the ones provided by the user to enum.